
git.enlightenment.org/enlightenment/efl/issues/71
git.enlightenment.org/enlightenment/efl/commit/6c3630f


From 6c3630ffda0884b86e6ffc9d00d7315ab67858e5 Mon Sep 17 00:00:00 2001
From: Ted Rodgers <ted.d.rodgers@gmail.com>
Date: Wed, 21 Aug 2024 10:56:13 -0400
Subject: [PATCH] eina_string_view.hh change lenght to length

fixes #71
---
 src/bindings/cxx/eina_cxx/eina_string_view.hh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/bindings/cxx/eina_cxx/eina_string_view.hh b/src/bindings/cxx/eina_cxx/eina_string_view.hh
index 77798db70f..f0bbcb705d 100644
--- a/src/bindings/cxx/eina_cxx/eina_string_view.hh
+++ b/src/bindings/cxx/eina_cxx/eina_string_view.hh
@@ -181,7 +181,7 @@ public:
    size_type rfind(basic_string_view<CharT, Traits> const& s) const
    {
       const_reverse_iterator iter = std::search(crbegin(), crend(), s.crbegin(), s.crend(), Traits::eq);
-      return iter == crend() ? npos : reverse_distance(crbegin(), iter) - s.lenght();
+      return iter == crend() ? npos : reverse_distance(crbegin(), iter) - s.length();
    }
 
    size_type rfind(basic_string_view<CharT, Traits> const& s, size_type pos) const
@@ -189,7 +189,7 @@ public:
       if (pos >= _len)
         return npos;
       const_reverse_iterator iter = std::search(crbegin()+pos, crend(), s.crbegin(), s.crend(), Traits::eq);
-      return iter == crend() ? npos : reverse_distance(crbegin(), iter) - s.lenght();
+      return iter == crend() ? npos : reverse_distance(crbegin(), iter) - s.length();
    }
 
    size_type rfind(CharT c) const

